target/linux/ath79/base-files/etc/board.d/02_network

```diff
@@ -71,15 +84,11 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "1:lan:1" "3:lan:4" "4:lan:3" "5:lan:2" "2:wan"
 		;;
-	comfast,cf-e5|\
+	comfast,cf-e110n-v2|\
+	comfast,cf-e120a-v3|\
 	glinet,gl-ar150|\
-	glinet,gl-ar300m-nand|\
-	glinet,gl-ar300m-nor|\
```

Comment:
gl-ar300m-nand and glinet,gl-ar300m-nor are only deleted, but not recreated.

Reply:
Both of them also uses the the last default case.